package-lock.json

```diff
@@ -3,6 +3,5444 @@
   "version": "0.0.1",
   "lockfileVersion": 1,
   "requires": true,
+  "packages": {
```

Comment:
Could you completely revert any changes to this file?

Reply:
Done!. I thought that only the  "lockfileVersion" was changed to 2, but I can see lots of changes...,  (no idea why that file was changed, I did 'nothing'... 🤷‍♂️)